docroot/modules/custom/foia_file/drush/foia_file.drush.inc

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * @file
+ * FOIA Api module drush command.
+ */
+
+use Drupal\file_entity\Entity\FileEntity;
+
+/**
+ * Implements hook_drush_command().
+ */
+function foia_file_drush_command() {
+  $items = [];
+
+  $items['file-entity-update'] = [
+    'description' => 'Updates file entities virus scan statuses.',
+    'core' => ['8+'],
+    'arguments' => [
+      'scan_result' => 'The result of the file scan.',
+    ],
+    'aliases' => ['feu'],
+  ];
+
+  return $items;
+}
+
+/**
+ * Implements hook_drush_help().
+ */
+function foia_file_drush_help($command) {
+  switch ($command) {
+    case 'drush:file-entity-update':
+      return dt('Update virus scan statuses for file entites based upon output from virus scanner.');
+  }
+}
+
+/**
+ * Implements drush_hook_COMMAND().
+ */
+function drush_foia_file_file_entity_update($scanOutput) {
+  $startTime = microtime(TRUE);
+
+  // Get webform directory on the server.
+  $fileDir = explode('/webform/', $scanOutput, 2);
+  $webformDir = "{$fileDir[0]}/webform/";
+
+  // Temporarily delete similar paths of the webform dir during processing.
+  $trimmedFiles = str_replace($webformDir, '', $scanOutput);
+
+  // Split string into lines. @todo change to preg split \R?
+  $scans = explode("\n", $trimmedFiles);
+
+  foreach ($scans as $scan) {
+
+    // Split the scan line into filename and status.
+    preg_match('/[^ ]*$/', $scan, $wordsArray);
+    // Set status to the last word of the line.
+    $status = $wordsArray[0];
+
+    $validStatuses = ['OK', 'Removed.', 'Empty file'];
+    if (!in_array($status, $validStatuses)) {
+      \Drupal::logger('foia_file')->critical("An invalid status was detected in the virus scan output.");
```

Comment:
Should we be considering this a critical error, or even invalid at all? I'm thinking there could be another status that we don't know about that is valid, but unexpected. The only thing we could say about it for sure is that it's `unexpected`, but without actually seeing what the scanner spit out, it's difficult to classify it as invalid or valid.